tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,15 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
+      it('#5471 PouchDB.plugin() should throw error if passed wrong type or empty object', function () {
+        PouchDB.plugin('pouchdb-adapter-memory').then(function () {
```

Comment:
you need to use a JavaScript try/catch rather than promises here. I think Mocha can do this for you - something like:

```
assert.throws(function() {
      PouchDB.plugin('pouchdb-adapter-memory');
}, Error, 'Invalid plugin: object passed in is empty or not an object');
```